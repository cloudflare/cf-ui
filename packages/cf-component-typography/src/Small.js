import React, { PropTypes } from 'react';

class Small extends React.Component {
  render() {
    return (
      <small className="cf-small">
        {this.props.children}
      </small>
    );
  }
}

Small.propTypes = {
  children: PropTypes.node
};

export default Small;
