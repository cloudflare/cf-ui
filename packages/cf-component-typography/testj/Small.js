import React from 'react';
import renderer from 'react-test-renderer';
import Small from '../src/Small';

describe('Small', () => {
  test('should render', () => {
    const component = renderer.create(<Small>Small</Small>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
