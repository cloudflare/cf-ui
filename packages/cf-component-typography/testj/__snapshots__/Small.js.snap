// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Small should render 1`] = `
<small
  className="cf-small"
>
  Small
</small>
`;
